### ISSUE TITLE
Randomly Disables and have to reload page for UBlock Origin to work

### ISSUE THREAD

**Issue by jblack68** [el1t/uBlock-Safari#38]
> Read first: <https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md>

### One or more specific URLs where the issue occurs

Dailyrecord.co.uk

### Your settings

> [If you fail to provide this info, I will mark the issue as invalid. Lists all settings which differs from default settings]

- Safari version:  10.1
- uBlock Origin version: 1.12.0 (b)
- MacOS version: 10.12.4

##### Your filter lists

"Default filter lists + Fanboy+Easylist-Merged Ultimate List

**Comment by freediverx** [comment]
This happens to me all the time as well, but it’s not consistent. I just clicked on your link and the page loaded without disabling uBlock.

**Comment by jblack68** [comment]
Yeah, it happens randomly and is impossible to reproduce as its not consistent.

Wonder if it can be fixed :(

**Comment by freediverx** [comment]
Not to flaunt my tinfoil hat, but I wouldn’t be surprised if this is due to some clever javascript used by websites to disable ad blockers. Even if you can override it by refreshing the page, the ads and trackers already got loaded the first time, so it’s a win for the webmaster and ad networks.

**Comment by el1t** [comment]
Sorry, but I'll need some more concrete steps to reproduce other than just browsing. Are you sure none of those sites are whitelisted?

**Comment by jblack68** [comment]
Nah, none of the sites are whitelisted, it will simply just deactivate randomly and have to reload page for the blocker to be active again.

**Comment by freediverx** [comment]
I usually encounter this issue when following a link to a story. Could this behavior depend on the referral link or some embedded code within?

**Comment by freediverx** [comment]
Here’s an example, but it doesn’t happen consistently. Visit https://www.reddit.com/r/apple/comments/66hslb/apple_produces_half_a_gallon_of_fake_sweat_every/ <https://www.reddit.com/r/apple/comments/66hslb/apple_produces_half_a_gallon_of_fake_sweat_every/> and then click on the story link pointing to a page on cultofmac.com.

**Comment by kode54** [comment]
I've had it happen recently while browsing random sites. I even had it disable itself on YouTube once.
